app/models/asset_field_type.rb

```diff
@@ -52,6 +53,29 @@ def mapping
 
   private
 
+  def is_image(mime_type)
+    ['image/jpeg', 'image/pjpeg', 'image/png','application/pdf' ,'image/x-png', 'image/gif'].include?(mime_type)
+  end
+
+  def format_data(metadata_hash)
+   return metadata_hash unless is_image(metadata_hash[:content_type])
+   metadata_hash.merge(image_asset_styles)
+  end
+
+  def image_asset_styles
+    {
+      styles: {
```

Comment:
So part of the reason that styles existed in media.rake was that it allowed the User to specify which size uploads they want, which does vary from case to case (especially things like the names of the styles - post_tile is a pretty CB specific concern).